strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/providers/repository/MavenGroupRepositoryProviderTest.java

```diff
@@ -262,7 +282,7 @@ public void testGroupIncludesWildcardRule(@MavenRepository(repositoryId = REPOSI
                                               @MavenTestArtifact(repositoryId = REPOSITORY_RELEASES_TGIWR_2, id = "com.artifacts.in.releases.under2:group", versions = "1.2.4") Path a4)
             throws Exception
     {
-        System.out.println("# Testing group includes with wildcard...");
+        logger.info("# Testing group includes with wildcard...");
```

Comment:
I think that in the test cases we should use `logger.debug`, so that we can fine-tune this, if we'd like to reduce the noise. For example, at the moment, our build are failing under Travis (where we run our MacOS builds) because the log files are exceeding the limit.